W:edge1

(rationale: see preceding patch in the series)

--- a/dd2.c
+++ b/dd2.c
@@ -15878,9 +15878,9 @@
         }
         piVar4 = (int *)(local_24 + iVar5 * 0xc);
         piVar3 = (int *)(iVar2 * 0xc + local_24);
-        local_70 = *piVar4 - *piVar3;
-        local_6c = piVar4[1] - piVar3[1];
-        local_68 = piVar4[2] - piVar3[2];
+        _e1[0] = *piVar4 - *piVar3;
+        _e1[1] = piVar4[1] - piVar3[1];
+        _e1[2] = piVar4[2] - piVar3[2];
         piVar4 = (int *)(iVar6 * 0xc + local_24);
         local_60 = *piVar4 - *piVar3;
         local_5c = piVar4[1] - piVar3[1];
